ui/components/institutional/institutional-entity-done-page/index.scss

```diff
@@ -0,0 +1,24 @@
+.institutional-entity-done {
+  &__container {
+    border-radius: 0;
+  }
+
+  &__form {
+    .pulse-loader {
+      justify-content: center;
+    }
```

Comment:
What element/component does this apply to? I don't see PulseLoader in this PR

Reply:
It's coming from here, is there a better way to do this?

![image](https://user-images.githubusercontent.com/1182864/227164098-a83c5b67-0769-4ba3-9fa1-448e10a4ed65.png)